app/react/Library/components/TableCell.tsx

```diff
@@ -4,15 +4,19 @@ import { I18NLink } from 'app/I18N';
 import GeolocationViewer from 'app/Metadata/components/GeolocationViewer';
 import { LinkSchema, MetadataObjectSchema, PropertySchema } from 'shared/types/commonTypes';
 import MarkdownViewer from 'app/Markdown';
+import { showByType } from 'app/Metadata/components/Metadata';
 
 export interface TableCellProps {
   content?: FormattedMetadataValue;
   zoomLevel: number;
 }
 
-export interface FormattedMetadataValue extends PropertySchema {
+export type FormattedMetadataValue = Omit<PropertySchema, 'type'> & {
```

Comment:
Can you please export `FormattedMetadataValue` & `TabelCellProps` at the end of the file?

Reply:
Done